### PR TITLE
Apacdex Bid Adapter: add support for meta.advertiserDomains, add and update sample adunit, validate dealId field from server response, add support Price Floors Module

### PR DESCRIPTION
**Pull request by thuyhq**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change
- Add meta.advertiserDomains to Apacdex bid adapter according to the issue mentioned https://github.com/prebid/Prebid.js/issues/6650
- Add and update sample ad units
- Validate dealId from server response
- Lint code
- Add support Price Floors Module according to the issue mentioned https://github.com/prebid/Prebid.js/issues/6465

### CODE REVIEW

**Comment by thuyhq** [comment]
Hello @ChrisHuie,
Looks like the error from circleCI is not coming from my code. What do I need to do to bypass it?
Thanks very much!

**Comment by patmmccann** [comment]
Remerge master, #6706 should help

File: modules/apacdexBidAdapter.md [hunk 1]
```diff
@@ -26,29 +26,77 @@ var adUnits = [
...
+      placement: 1,
+      skip: 1,
+      linearity: 1,
+      minduration: 1,
+      maxduration: 120,
+      mimes: ["video/mp4", "video/x-flv", "video/x-ms-wmv", "application/vnd.apple.mpegurl", "application/x-mpegurl", "video/3gpp", "video/mpeg", "video/ogg", "video/quicktime", "video/webm", "video/x-m4v", "video/ms-asf", video/x-msvideo"],
+      playbackmethod: [6],
+      startdelay: 0,
+      protocols: [1, 2, 3, 4, 5, 6]
+      ... // Aditional ORTB video params
```
Comment (patmmccann) [inline modules/apacdexBidAdapter.md:56]: spelling

File: modules/apacdexBidAdapter.md [hunk 2]
```diff
@@ -26,29 +26,77 @@ var adUnits = [
...
+      // you must review all video parameters to ensure validity for your player and DSPs
     },
   },
   bids: [
     {
       bidder: 'apacdex',
       params: {
         siteId: 'apacdex1234', // siteId provided by Apacdex
+        floorPrice: 0.01, // default is 0.01 if not declared
```
Comment (patmmccann) [inline modules/apacdexBidAdapter.md:65]: not mentioned in your js.

File: modules/apacdexBidAdapter.md [hunk 3]
```diff
@@ -26,29 +26,77 @@ var adUnits = [
...
+      ... // Aditional ORTB video params
+      // you must review all video parameters to ensure validity for your player and DSPs
+    },
+  },
+  bids: [
+    {
+      bidder: 'apacdex',
+      params: {
+        siteId: 'apacdex1234', // siteId provided by Apacdex
+        floorPrice: 0.01, // default is 0.01 if not declared
```
Comment (patmmccann) [inline modules/apacdexBidAdapter.md:99]: not in the js

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/apacdexBidAdapter.md (hunk 2 above)
Comment (patmmccann) [reply modules/apacdexBidAdapter.md:65]: if you plan to add it, please note #6465

**Comment by thuyhq** [comment]
Hello @patmmccann,
When will my PR be merged?
Is there anything I can do right now?

**Review by patmmccann — COMMENTED** [review]
Lgtm

**Pull request merged**